main.cc

```diff
@@ -301,7 +301,10 @@ init()
     frame_index = 0;
 
     pl.rot = glm::quat(1, 0, 0, 0);
+    auto m = glm::mat4_cast(glm::normalize(pl.rot));
```

Comment:
We saw all kinds of bad behavior in initial bullet state already -- is this why?